### PR TITLE
Make server launchable from a central point

### PR DESCRIPTION
**Pull request by luisferreira32**
The objective of this PR is to enable a hybrid application entry point. If we launch from a service/ we can do a monolithic app, whereas, launching from micro-services is still possible.

### CODE REVIEW

**Comment by luisferreira32** [comment]
Note, did some filename renames to have it at the top of the file tree (underscore comes first).
Also do some relative imports so we don't have to fight the PYTHONPATH.

File: services/server.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import uvicorn
+import matplotlib
+
+from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
+from decouple import config, Csv
+
+from services.plotter.server import plotter_router
+# from services.example_service.server import example_router
```
Comment (luisferreira32) [inline services/server.py:9]: Left some examples for future additions of micro services to the monolith (and yes, sentence with contradictions)

File: services/plotter/plotter.py [hunk 2]
```diff
@@ -1,31 +0,0 @@
-from io import BytesIO
```
Comment (luisferreira32) [inline services/plotter/plotter.py:1]: thought this file did not need exist 💥

File: services/server.py (hunk 1 above)
Comment (MrToino) [reply services/server.py:9]: > Note, did some filename renames to have it at the top of the file tree (underscore comes first). Also do some relative imports so we don't have to fight the PYTHONPATH.

That's great!

File: services/fitter/service.py [hunk 3]
```diff
@@ -2,8 +2,8 @@
 
 from fitter import fitter
 
-from utils.log import logger
-from utils.preprocessor import validate_data
+from .._utils.log import logger
```
Comment (MrToino) [inline services/fitter/service.py:5]: This is good, although I'm not sure if that's the best practice. Maybe in the future, we can use something as `sys.path.append` to append the working directory to the path ([see this](https://stackoverflow.com/questions/9383014/cant-import-my-own-modules-in-python)).

**Review by MrToino — APPROVED** [review]
Seems all good, just please check the review before merging.

File: services/fitter/service.py (hunk 3 above)
Comment (luisferreira32) [reply services/fitter/service.py:5]: `sys.path.append` is not the way, for sure... He even says it's not recommended :sweat_smile: 
We could do some setups with PYTHONPATH for development, but would have to create some way to actually do packaged modularization of our microservices like suggested in the lower parts of the answer...?

**Pull request merged**